hubspot/read.go

```diff
@@ -19,7 +19,7 @@ func (c *Connector) Read(ctx context.Context, config common.ReadParams) (*common
 
 	// If filtering is required, then we have to use the search endpoint.
 	if requiresFiltering(config) {
-		return c.search(ctx, config)
+		return nil, ErrFilteringNotSupported
```

Comment:
See comment above, instead of returning an error, we should be constructing a search query, the comment above the function signature in line 14 should specify that if `Since` is present, then result is limited to 10,000 records. If more records are desired, then the Search function should be used.